cherab/generomak/plasma/plasma.py

```diff
@@ -419,15 +390,23 @@ def get_core_profiles_description(lcfs_values=None, core_args=None):
         z = 0
         lcfs_values = get_edge_profile_values(r, z)
 
+        # manually correcting the LCFS densities for neutral species and low ionised carbon
+        # to better match the edge profiles
+        lcfs_values["composition"]["hydrogen"][0]["density"] = 1.e15
+        lcfs_values["composition"]["carbon"][0]["density"] = 1.e5
+        lcfs_values["composition"]["carbon"][1]["density"] = 1.5e5
+        lcfs_values["composition"]["carbon"][2]["density"] = 1.e9
+        lcfs_values["composition"]["carbon"][3]["density"] = 1.e13
+
```

Comment:
Can we correct these values at source, rather than hidden away here?

Reply:
Just realized that there is a more serious problem here, the core profiles do not match plasma neutrality. We need to use the ionisation balance solver. I'll try to fix this.